tools/src/scie_pants/install_pants.py

```diff
@@ -67,8 +67,58 @@ def pip_install(*args: str) -> None:
     #
     # Also, we don't advance setuptools past 58 which drops support for the `setup` kwarg `use_2to3`
     # which Pants 1.x sdist dependencies (pystache) use.
-    pip_install("-U", "pip==22.3.1", "setuptools<58", "wheel")
-    pip_install("--progress-bar", "off", *pants_requirements)
+    venv_pip_install(venv_dir, "-U", "pip==22.3.1", "setuptools<58", "wheel", find_links=find_links)
+    venv_pip_install(venv_dir, "--progress-bar", "off", *pants_requirements, find_links=find_links)
+
+
+def install_pants_from_pex(
+    venv_dir: Path,
+    prompt: str,
+    version: Version,
+    ptex: Ptex,
+    extra_requirements: Iterable[str],
+    find_links: str | None,
+) -> None:
+    """Installs Pants into the venv using the platform-specific pre-built PEX."""
+    uname = os.uname()
+    major, minor = sys.version_info[:2]
+    pex_name = f"pants.cp{major}{minor}-{uname.sysname.lower()}_{uname.machine.lower()}.pex"
+    with tempfile.NamedTemporaryFile(suffix=".pex") as pants_pex:
+        pex_url = (
+            f"https://github.com/pantsbuild/pants/releases/download/release_{version}/{pex_name}"
+        )
+        try:
+            ptex.fetch_to_fp(pex_url, pants_pex.file)
```

Comment:
OK I think the last thing I'd love to see here is that the PEX is cached. If I have multiple Pants repos, but they use the same version, it'd be nice to have the PEX be cached.

But I'll call it a "nice-to-have" and just open an issue.